week-8-practise/3-classes.js

```diff
@@ -23,20 +23,98 @@
  * Using your newly created class
  *  - Instantiate (hint: new House())a brown house with 3 windows and 2 bedrooms that is based in Woking
  *  - Instantiate another blue house with 5 windwos and 2 bedrooms that is based in London
- *  - Return and compare the price of the two houses
+ *  - Return and compare the price of the two housese value of the house should increase by 80%
  *  - Return and compare the energy bill of the two houses
  *  - Return how many square meters are in 950 square feet
  */
 
 class House {
-  constructor(windows, colour) {
-    this.windows = windows
+  constructor(windows, colour = "white", numberOfRooms, location) {
```

Comment:
We haven't covered this in class yet, but default parameters should always be last (JS maps values to the parameters by position, left to right)

Reply:
Thanks @sansaid  for your review I have updated and added the the default parameter.